benchmarks/benchmarks/bench_clip.py

```diff
@@ -0,0 +1,19 @@
+from .common import Benchmark
+
+import numpy as np
+
+
+class ClipFloat(Benchmark):
+    param_names = ["dtype", "size"]
+    params = [
+        [np.float32, np.float64],
+        [100_000]
```

Comment:
```suggestion
        [100, 100_000]
```
Also check performance on smaller size arrays. (for this PR I am sure the performance is not affected)

Reply:
I have added the 100 and will run the benchmark again later.

I did have sizes 1000 and 10_000 originally and I recall 1000 was not measurably faster.
The sizes that are interesting to me are > 1_000_000. (rescaling images)